pulp-glue/pulp_glue/common/context.py

```diff
@@ -120,7 +120,7 @@ def __init__(
         api_root: str,
         api_kwargs: Dict[str, Any],
         background_tasks: bool,
-        timeout: int,
+        timeout: Union[int, datetime.timedelta],
```

Comment:
I like the change overall. But, can you provide an example where `datetime.timedelta` will be used?

Reply:
Well it's an extension to the current interface, currently used nowhere. It felt like a low hanging fruit to me.